main-app/app/scripts/bingoNumbers/callingNumbers.js

```diff
@@ -0,0 +1,45 @@
+(function () {
+    'use strict';
+    angular.module('Tombola.LongPollingService')
+        .service('longPolling',['$timeout', 'proxy', 'userModel', 'numberService', function ($timeout, proxy, userModel, numberService) {
+            var me = this;
+            me.currentNumber = 0;
```

Comment:
Very few of these values are related to polling - can't just whack it all into one service.
- Numbers to win is a responsibility of the ticket/game not the polling infrastructure.